src/lib/core/data-table/data-table.ts

```diff
@@ -161,7 +145,7 @@ export class CdkTable<T> implements CollectionViewer {
   }
 
   ngOnDestroy() {
-    this._onDestroy.next();
+    this._onDestroy.complete();
```

Comment:
Thanks for the heads up, I appreciate the plunker to help me understand. Looking at the [RxJS code](https://github.com/Reactive-Extensions/RxJS/blob/master/src/core/perf/operators/takeuntil.js) does indicate that it doesn't look to the completion event.

`Returns the values from the source observable sequence until the other observable sequence produces a value.`

This doesn't line up with the documentation from reactivex.io which says that either an emitted value or termination notification would end it.

Will change it to be next() and complete(). Not totally sure the complete() is necessary but we can discuss another time.